src/lib/components/ui/hr.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+	import type { HTMLAttributes } from 'svelte/elements'
+
+	interface Props extends HTMLAttributes<HTMLHRElement> {
+		class?: string
+	}
+	let { class: className = '', ...restProps }: Props = $props()
+</script>
+
+<hr class={className} {...restProps} />
```

Comment:
This can be simplified as there is no need to extract `class` from props
```suggestion
	let props: HTMLAttributes<HTMLHRElement> = $props()
</script>

<hr {...props} />
```